JASP-Engine/JASP/R/principalcomponentanalysis.R

```diff
@@ -677,65 +731,13 @@ PrincipalComponentAnalysis <- function(dataset = NULL, options, perform = "run",
 }
 
 # Goodness of fit
-.goodnessOfFitPCA <- function(analysisResults, options,perform){
-
-  # Extract loadings:
-  if (!is.null(analysisResults)){
-    
-#     print(analysisResults)
-#     print(analysisResults$RMSEA)
-#     
-#     Fits <- list(
-#       CHI = analysisResults$STATISTIC,
-#       PVAL = analysisResults$PVAL,
-#       DF = analysisResults$dof,
-#       RMSEA = analysisResults$RMSEA['RMSEA'],
-#       RMSEAlower = analysisResults$RMSEA['lower'],      
-#       RMSEAupper = analysisResults$RMSEA['upper'],      
-#       TLI = analysisResults$TLI,
-#       RMS = analysisResults$rms,
-#       CRMS = analysisResults$crms,
-#       BIC = analysisResults$BIC
-#     )
+.goodnessOfFit <- function(analysisResults, options, perform){
```

Comment:
@hummyhummy common function

Reply:
this is a common function only in file principalcomponentanalysis.R